src/QMCWaveFunctions/Fermion/MultiSlaterDeterminantFast.h

```diff
@@ -132,6 +132,10 @@ class MultiSlaterDeterminantFast : public WaveFunctionComponent
                            std::vector<ValueType>& dlogpsi,
                            std::vector<ValueType>& dhpsioverpsi);
 
+  void evaluateDerivativesWF(ParticleSet& P,
```

Comment:
Please use override key word to explicitly overriding the virtual function. This should help us writing better code.
evaluateDerivativesWF() override
Only needed for the derived class function declaration. Not needed in the definition.
Please mark all the overriding functions in this class.

Reply:
marked with override keyword.